apps/src/templates/exportProject.html.ejs

```diff
@@ -2,7 +2,8 @@
   <head>
     <title>My App</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <script src="applab.js"></script>
+    <script src="https://code.jquery.com/jquery-3.1.1.min.js"></script>
```

Comment:
Our site currently uses jQuery 1.12.1: We are [using jquery-rails 4.1.1](https://github.com/code-dot-org/code-dot-org/blob/staging/Gemfile.lock#L307) which [loads jQuery 1.12.1](https://github.com/rails/jquery-rails/blob/master/CHANGELOG.md#411), which can be verified in the console:

![image](https://cloud.githubusercontent.com/assets/1615761/20682436/ec691630-b55c-11e6-8f65-f668a4b8a84a.png)

As out-of-date as that is, it seems like a difference of two major versions might cause problems.  Maybe we should use 1.12.1 here?

```
<script src="http://code.jquery.com/jquery-1.12.1.min.js" integrity="sha256-I1nTg78tSrZev3kjvfdM5A5Ak/blglGzlaZANLPDl3I=" crossorigin="anonymous"></script>
```